### PR TITLE
ubuntu impish

### PR DESCRIPTION
**Pull request by romanyacik**
entry added for Ubuntu Impish

Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [ ] This is a documentation change only

Issues:
- Please list any related issues

Purpose:
- What is this pull request trying to do?
- What release is this for?
- Is there a project or milestone we should apply this to?

### CODE REVIEW

File: .github/workflows/gh-actions-pr.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ jobs:
       fail-fast: true
       matrix:
         include:
+          - FROM:     'ubuntu:impish'
+            COMPILER: 'gcc'
+            PREFER_LEGACY_OPENGL: 'OFF'
```
Comment (BenjamenMeyer) [inline .github/workflows/gh-actions-pr.yml:20]: @romanyacik we also need a Clang version as we use multiple compilers. Use the `hirsute` as an example.
@stephengtuggy do we need both version of PREFER_LEGACY_OPENGL? Or only one version?

**Review by BenjamenMeyer — CHANGES_REQUESTED** [review]
@romanyacik  thanks for submitting this; I do have a slight change request though you may want to wait until @stephengtuggy gives some feedback first.

File: .github/workflows/gh-actions-pr.yml (hunk 1 above)
Comment (stephengtuggy) [reply .github/workflows/gh-actions-pr.yml:20]: Yes, I would say we need all four combinations of gcc vs. clang, and PREFER_LEGACY_OPENGL `ON` vs. `OFF`.

Before any of that will work, though, we need to add Impish to the bootstrap scripts; to CMakeLists.txt; and to the CI/CD for `build-system-docker-images`.

**Comment by stephengtuggy** [comment]
Can we close this as superseded by #625 ?

**Comment by stephengtuggy** [comment]
@romanyacik thanks for the impetus for Impish support. That will be good to have in 0.8.x and following.

**Comment by BenjamenMeyer** [comment]
Closing due to age and superceded by #625 per https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/612#issuecomment-1000963282

**Pull request closed without merging**